server/models/Interview.js

```diff
@@ -44,19 +44,16 @@ const InterviewSchema = new Schema(
 );
 
 async function createInterview(req) {
-  const { creator } = req.body;
-  const firstUser = await User.findOne({ _id: creator });
-
+  const firstUser = req.user;
   try {
-    const interview = new Interview({});
-    const interviewDoc = await interview.save();
+    const interviewDoc = await new Interview({}).save();
```

Comment:
does it need to be saved first for this to work? saving twice might be unnecessarily expensive database operation